src/elements/button/button.svelte

```diff
@@ -54,15 +52,16 @@ const handleParentClick = (e: PointerEvent) => {
   :host { display: inline-block !important }
 </style>
 
+<!-- svelte-ignore a11y-click-events-have-key-events -->
 <svelte:element
   this={tooltip ? 'v-tooltip' : 'span'}
   text={tooltip}
-  on:click={handleParentClick}
+  on:click={isDisabled ? handleParentClick : undefined}
 >
   <button
     {type}
     aria-label={variant === 'icon' ? label : undefined}
-    aria-disabled={isDisabled}
+    disabled={isDisabled}
```

Comment:
Hm, this will mean the buttons will not be tab-navigable and won't be announced by assistive technology. It will essentially hide the button from them entirely, but I can imagine cases where a PRIME user would want a disabled button to still be recognizable/navigable.

Reply:
Yeah, this is a good point. Thoughts on keeping `aria-disabled` for button and `disabled` for inputs?

Reply:
That works for me.